src/components/VerificationTracker/VerifyID.js

```diff
@@ -101,6 +106,15 @@ const VerifyID = () => {
           />
         )}
       </div>
+      {userAddress &&
+        userAddress[0].label.toLowerCase() ===
+          "GPIB Custodial Wallet".toLowerCase() && (
```

Comment:
This could be abused because anyone can add a BTC address and label it "GPIB Custodial Wallet"

Reply:
> This could be abused because anyone can add a BTC address and label it "GPIB Custodial Wallet"

Good catch. We should probably make a change to the backend to be able to mark an address as a custodial wallet.

Reply:
using isCustodial